consensus/istanbul/backend/engine_test.go

```diff
@@ -72,6 +72,11 @@ func makeCommittedSeals(hash common.Hash) [][]byte {
 	return committedSeals
 }
 
+func stopBlockChain(bc *blockchain.BlockChain, backend *backend) {
+	backend.Stop()
+	bc.Stop()
```

Comment:
This might help when backend or bc is nil.
```suggestion
	if backend != nil {
		backend.Stop()
	}
	if bc != nil {
		bc.Stop()
	}
```